### DISCUSSION TITLE
第6节课笔记(19班)

### DISCUSSION THREAD

**Discussion by LYMDLUT** [InternLM/Tutorial#445] [笔记]
第6节课笔记(19班)

**Comment by AllYoung** [comment]
https://blog.csdn.net/sunshine_youngforyou/article/details/135718361

**Comment by Sixdes** [comment]
https://bbs.csdn.net/topics/617976995?spm=1001.2014.3001.6377

**Comment by jts250** [comment]
https://zhuanlan.zhihu.com/p/679189417

**Comment by romanwin28** [comment]
https://blog.csdn.net/George_han/article/details/135819020

**Comment by Jiangzzzzl** [comment]
https://blog.csdn.net/2301_80618119/article/details/135721550?spm=1001.2014.3001.5501

**Comment by melonwine** [comment]
https://github.com/melonwine/InternLMPlay/blob/master/06/note06.md

**Comment by mkdirmushroom** [comment]
[第6节课笔记](https://github.com/mkdirmushroom/note/blob/main/InternLM/tutorial5_note.md)

**Comment by Hamasuta666** [comment]
第6节课笔记（作业在文末）：https://zhuanlan.zhihu.com/p/681111451
